packages/eas-cli/src/build/android/version.ts

```diff
@@ -165,3 +169,53 @@ async function writeBuildGradleAsync({
   const buildGradlePath = AndroidConfig.Paths.getAppBuildGradleFilePath(projectDir);
   await fs.writeFile(buildGradlePath, buildGradle);
 }
+
+export async function updateToNextVersionCodeAsync({
+  projectDir,
+  projectId,
+  exp,
+  applicationId,
+  buildProfile,
+}: {
+  projectDir: string;
+  projectId: string;
+  exp: ExpoConfig;
+  applicationId: string;
+  buildProfile: BuildProfile<Platform.ANDROID>;
+}): Promise<string> {
+  const remoteVersions = await AppVersionQuery.latestVersionAsync(
+    projectId,
+    AppPlatform.Android,
+    applicationId
+  );
+
+  const localVersions = await maybeResolveVersionsAsync(projectDir, exp, buildProfile);
```

Comment:
Maybe move this function outside the `metadata.ts` module.

Reply:
not sure what you mean, this function is not defined in metadata.ts but there is a function there with the same name